web/v1/snowflake/sql-runner/sql/standard/00-setup/01-main/01-stored-procedures.sql

```diff
@@ -334,42 +300,41 @@ CREATE OR REPLACE PROCEDURE {{.output_schema}}.commit_with_metadata(
    - Imitates BigQuery's EXCEPT(col1,...) functionality.
    - Accepts a string of comma separated column tranformations to add
    - and another with their respective aliases.
-
   Input:
-  SRC_SCHEMA:    the schema of the source table
-  SRC_TBL:       the source table name
+  SOURCE_TABLE:  the source table
   XCLUDE_COLS:   a string of comma-separated columns to exclude
   TRANS_COLS:    a string of comma-separated column tranformations
   TRANS_ALIASES: a string of comma-separated aliases for the corresponding trans_cols
-  TRG_SCHEMA:    the schema of the target table
-  TRG_TBL:       the target table name
-
+  TARGET_TABLE:  the target table
+  FROM_CLAUSE:   the from clause of the ctas statement
+  WHERE_CLAUSE:  the where clause of the ctas statement
   Example call:
-  CALL ctas_similar(UPPER('scratch'),
-                    UPPER('base_events_this_run'),
-                    UPPER('contexts_com_snowplowanalytics_snowplow_web_page_1, my_unwanted_column'),
-                    'contexts_com_snowplowanalytics_snowplow_web_page_1[0]:id::varchar(36)',
-                    'page_view_id',
-                    UPPER('scratch'),
-                    UPPER('events_this_run);
-
+  CALL ctas_similar(UPPER('scratch.base_events_this_run'),
+                    UPPER('my_context'),
+                    'my_context:myKey::float',
+                    'my_new_alias',
+                    UPPER('scratch.events_this_run'),
+                    'FROM SCRATCH.TMP',
+                    '';
   To drop
   DROP PROCEDURE {{.output_schema}}.ctas_similar(VARCHAR,VARCHAR,VARCHAR,VARCHAR,
                                                  VARCHAR,VARCHAR,VARCHAR);
 
 */
-CREATE OR REPLACE PROCEDURE {{.output_schema}}.ctas_similar(SRC_SCHEMA    VARCHAR,
-                                                            SRC_TBL       VARCHAR,
-                                                            XCLUDE_COLS   VARCHAR,
-                                                            TRANS_COLS    VARCHAR,
-                                                            TRANS_ALIASES VARCHAR,
-                                                            TRG_SCHEMA    VARCHAR,
-                                                            TRG_TBL       VARCHAR)
+CREATE OR REPLACE PROCEDURE {{.output_schema}}.ctas_similar(
```

Comment:
This is great - there's good reason to keep it as a stored procedure. As we realised when we discussed, it can be defined without any inputs since it's only used once - which should improve readability.